src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -92,6 +92,13 @@ public function load(array $configs, ContainerBuilder $container)
             $dir = dirname($reflection->getFileName()).'/Resources/views';
             if (is_dir($dir)) {
                 $this->addTwigPath($twigFilesystemLoaderDefinition, $dir, $bundle);
+
+                $classMock = new $class();
```

Comment:
I'm afraid we cannot instantiate the bundle class like this. Some bundles might have overridden the constructor (like the [JMSDebuggingBundle](https://github.com/schmittjoh/JMSDebuggingBundle/blob/master/JMSDebuggingBundle.php#L33)).